core/constants_test.py

```diff
@@ -122,3 +122,9 @@ def test_language_constants_are_in_sync(self) -> None:
             set(rtl_audio_languages) & set(ltr_content_languages)
         )
         self.assertFalse(conflicts)
+
+    def test_constants_attr_can_be_changed(self) -> None:
+        """Test that constants may be changed through assignment."""
+        self.assertEqual(constants.constants.DEFAULT_LANGUAGE_CODE, 'en')
+        constants.constants.DEFAULT_LANGUAGE_CODE = 'zh'
```

Comment:
I think we prefer using `self.swap` here so that the changes are reversed once the test completes. @nithusha21 is that correct?

For info on how to use `self.swap` see https://github.com/oppia/oppia/wiki/Backend-tests#backend-test-structure

Reply:
Yup, we should do that